extras/repos/dave_sim.repos

```diff
@@ -1,65 +1,37 @@
 repositories:
-  dave/dave:
-    type: git
-    url: https://github.com/Field-Robotics-Lab/dave.git
-    version: master
-
-  dave/acoustic_msgs:
+  acoustic_msgs:
```

Comment:
Rename to `hydrographic_msgs`? The URL is being redirected there

Reply:
Yes - corrected in e8d1b987eff6e2978914f4db5789d8f5f6bd9b73